roles/openshift_logging/tasks/oc_apply.yaml

```diff
@@ -18,6 +18,26 @@
   failed_when: "'error' in generation_apply.stderr"
   changed_when: no
```

Comment:
Is there any way to do the check closer as the immediate next task? I worry about the case where the playbook is interrupted or there is a failure between this task an the change being reported...

Ideally, there would be a way to get the object back from apply to immediately validate the changed status in the same task.